src/main/scala/tech/cryptonomic/conseil/tezos/TezosRemotes.scala

```diff
@@ -0,0 +1,202 @@
+package tech.cryptonomic.conseil.tezos
+
+import tech.cryptonomic.conseil.config.{HttpStreamingConfiguration, NetworkCallsConfiguration}
+import tech.cryptonomic.conseil.config.Platforms.TezosConfiguration
+import tech.cryptonomic.conseil.generic.chain.RemoteRpc
+import tech.cryptonomic.conseil.util.JsonUtil.JsonString
+
+
+object TezosRemoteInstances {
+
+  object Akka {
```

Comment:
Could we give this a more descriptive name, e.g. `AkkaRemoteInstance`?

Reply:
I'm not sure it would help, as this "module objects" are named to make instances visible through a local import that would look like
`import TezosRemotesInstances.Akka.Futures._` or
`import TezosRemotesInstances.Akka.Streams._` or 
`import TezosRemotesInstances.Cats.IO._`

I'm not sure if a more descriptive name will help here
`import TezosRemotesInstances.AkkaRemoteInstance.FutureInstances._` or something alike...
what do you think?